specs/agents/mobile/events.md

```diff
@@ -0,0 +1,53 @@
+## Mobile Events
+
+This document describes event used by the Mobile SDKs using the [OpenTelementry Event Api](https://github.com/open-telemetry/opentelemetry-specification/blob/0a4c6656d1ac1261cfe426b964fd63b1c302877d/specification/logs/event-api.md).
+Event names will be recording using the `event.name`
+
+
+### `ApplicationLaunch` 
+This event represents the occurrence of app entering the foreground or background or other application lifecycle states.
+
+#### Attributes
+| Name         | Type   | Values                 | Description                                        |
+|--------------|--------|------------------------|----------------------------------------------------|
+| `event.name` | String | `ApplicationLifecycle` | The name of the event                              |
+| `state`      | String | `foreground`           | The lifecycle state the app is transitioning to.   | 
+| `state.last` | String | `background`           | The lifecycle state the app is transitioning from. | 
+
```

Comment:
Do we need additional fields for example to report the app launch time?

Reply:
This could be difficult due to how iOS captures app launch time.